mapie/classification.py

```diff
@@ -962,12 +985,15 @@ def fit(
 
         if self._target_type == "multiclass":
             if self.method == "raps":
-                X, self.X_raps, y_enc, self.y_raps = train_test_split(
-                    X,
-                    y_enc,
-                    test_size=size_raps,
-                    random_state=self.random_state
+                raps_split = ShuffleSplit(
+                    1, test_size=size_raps, random_state=self.random_state
```

Comment:
why this change ?

Reply:
I need to have access to the index list (for train and and val split).